### ISSUE TITLE
Overzealous whitespace removal in paths

### ISSUE THREAD

**Issue by davidar** [svg/svgo#1189]
Sometime in the past year, svgo has started removing more whitespace from paths than it used to (it looks like it may be since commit 258fecf?), in a way that breaks the SVG for some viewers. For example, with this image:

```svg
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="573.331" height="367.563" viewBox="0 0 429.998 275.672"><defs><symbol overflow="visible" id="c"><path d="M1.922-.828h3.437V0H.734v-.828c.375-.383.883-.906 1.532-1.563a52.138 52.138 0 0 0 1.218-1.265c.313-.352.532-.649.657-.89.125-.25.187-.493.187-.735 0-.383-.137-.696-.406-.938-.274-.25-.625-.375-1.063-.375-.312 0-.64.059-.984.172-.336.106-.7.266-1.094.484v-1a8.838 8.838 0 0 1 1.11-.359c.343-.082.66-.125.953-.125.75 0 1.347.188 1.797.563.457.375.687.882.687 1.515 0 .305-.058.59-.172.86-.117.261-.32.574-.61.937-.085.094-.343.371-.78.828-.438.45-1.055 1.078-1.844 1.89zm0 0"/></symbol></defs><g transform="translate(-11.094 -12.11)"><use height="100%" width="100%" y="208.938" x="16.004" xlink:href="#c"/></g></svg>
```

svgo v1.3.2 modifies the path in these places:

```
Before: a52.138 52.138 0 0 0 1.218-1.265
After:  a52.138 52.138 0 001.218-1.265
Before: a8.838 8.838 0 0 1 1.11-.359
After:  a8.838 8.838 0 011.11-.359
```

Some viewers (e.g. Firefox) seem to be ok with this, but others (e.g. `rsvg-convert`) are not:

Before:

![Screenshot_20191111_173105](https://user-images.githubusercontent.com/24291/68569110-0a7a6900-04a9-11ea-9a0c-2942755fadf3.png)

After:

![Screenshot_20191111_173027](https://user-images.githubusercontent.com/24291/68569080-f3d41200-04a8-11ea-9d8a-dd907c1d7c6d.png)

**Comment by davidar** [comment]
Oops, looks like this is a duplicate of #1137 and #1164
